net-misc/ptpd/files/ptpd-2.3.0-ntpdc.patch

```diff
@@ -0,0 +1,22 @@
+diff -urN ptpd-2.3.0.old/src/dep/ntpengine/ntpdcontrol.c ptpd-2.3.0.new/src/dep/ntpengine/ntpdcontrol.c
```

Comment:
A generic question here: did you submit this patch upstream? Since it looks like a genuine bugfix :)